rules/windows/registry_event/sysmon_uac_bypass_shell_open.yml

```diff
@@ -14,10 +16,18 @@ logsource:
     category: registry_event
     product: windows
 detection:
-    selection:
+    selection1:
         TargetObject|endswith: '_Classes\ms-settings\shell\open\command\SymbolicLinkValue'
         Details|contains: '\Software\Classes\{'
-    condition: selection
+    selection2:
+        TargetObject|endswith: '_Classes\ms-settings\shell\open\command\DelegateExecute'
+    selection3:
+        TargetObject|endswith: '_Classes\ms-settings\shell\open\command\(Default)'
+    selection4:
+        TargetObject|endswith: '_Classes\exefile\shell\open\command\(Default)'
```

Comment:
Can be merge : 
```yaml
    selection3:
        TargetObject|endswith:
            - '_Classes\ms-settings\shell\open\command\(Default)'
            - '_Classes\exefile\shell\open\command\(Default)'
     condition: selection1 or selection2 or ( selection3 and not filter)
```
The filter is only for selection3 so why not a ~"filter_sel3" name ?